src/aliceVision/photometricStereo/normalIntegration.cpp

```diff
@@ -543,10 +543,10 @@ void adjustScale(const sfmData::SfMData& sfmData, image::Image<float>& initDepth
         const sfmData::Landmark& currentLandmark = landmarks.at(currentLandmarkIndex);
         knownDepths(i) = pose.depth(currentLandmark.X);
 
-        sfmData::Observation observationInCurrentPicture = currentLandmark.observations.at(viewID);
+        sfmData::Observation observationInCurrentPicture = currentLandmark.getObservations().at(viewID);
 
-        int rowInd = observationInCurrentPicture.x(1);
-        int colInd = observationInCurrentPicture.x(0);
+        int rowInd = observationInCurrentPicture.getCoordinates()(1);
```

Comment:
Why not using getX() and getY() instead of getCoordinates()(0) and getCoordinates()(1)